src/shipit_signoff/docs/architecture.gv

```diff
@@ -0,0 +1,21 @@
+digraph {
+	shipit_signoffs [label="Signoffs Service" shape=box]
+		shipit_signoffs -> db
+		shipit_signoffs -> balrog_admin_api
+		shipit_signoffs -> shipit_notifications
+	shipit_pipeline [label="Pipeline Service" shape=box]
+		shipit_pipeline -> shipit_signoffs
+	shipit_frontend [label="Ship It UI" shape=box]
+		shipit_frontend -> shipit_pipeline
+		shipit_frontend -> shipit_signoffs
+	user [label=User shape=box]
+		user -> shipit_frontend
+		user -> auth0
+		user -> "login.taskcluster.net"
+	shipit_notifications [label="Notifications Service" shape=box]
+	db [label=db shape=box]
+	balrog_admin_api [label="Balrog Admin API" shape=box]
+	auth0 [label=Auth0 shape=box]
+		auth0 -> shipit_signoffs
+	"login.taskcluster.net" [label="login.taskcluster.net" shape=box]
+}
```

Comment:
Could we use [sphinx.ext.graphviz](http://www.sphinx-doc.org/en/stable/ext/graphviz.html#module-sphinx.ext.graphviz) instead and not include generated images but generating them in the build process.

It might even make sense to use [sequential diagram](http://blockdiag.com/en/seqdiag/examples.html) for architecture. I find current graph not really explaining it well.